Formplayer/testCases/conftest.py

```diff
@@ -0,0 +1,161 @@
+import os
+import pytest
+
+from configparser import ConfigParser
+from pathlib import Path
+from selenium import webdriver
+from webdriver_manager.chrome import ChromeDriverManager
+from webdriver_manager.firefox import GeckoDriverManager
+
+from Formplayer.userInputs.user_inputs import UserData
+from Formplayer.testPages.base.login_page import LoginPage
+
+""""This file provides fixture functions for driver initialization"""
+
+
+global driver
+
+
+@pytest.fixture(scope="session")
+def environment_settings():
```

Comment:
Defining fixtures in the root `conftest.py`  is shared among all tests in your test suite, and it would possibly slow down testing if such fixtures are not used by all tests. But yes, we explore options here.

Reply:
https://dimagi-dev.atlassian.net/browse/QA-4401